lib/chef/config.rb

```diff
@@ -643,6 +643,12 @@ def self.windows_home_path
     default :normal_attribute_whitelist, nil
     default :override_attribute_whitelist, nil
 
+    config_context :windows_service do
+      # Set `watchdog_timeout` to the number of seconds to wait for a chef-client run
+      # to finish
+      default :watchdog_timeout, 20 * 60
```

Comment:
This seems pretty low to me. Is there any risk with setting to to something like 2 hours?